data/json/mapgen/police_department.json

```diff
@@ -123,9 +123,9 @@
         "|||||||||||||_.._|4::::::::::::::::|X:|_..i|||||",
         "|!|!|yyyyyyy|_.._|4::4444::4|222222|::|_..i|SSS|",
         "|___|_______|_.._|4::4││4::4||||||||Y:|_.._|___|",
-        "|___+_qqqqq_+_.._|4::4││4::4|{}[[[[|Y:|_.._+__a|",
```

Comment:
Police departments were a recurring problem, especially with respect to shotguns.
They're also pretty crazy common so I had to dial that down a bit too.